website/src/pages/DailyPage/DailyPage.jsx

```diff
@@ -222,34 +215,26 @@ const Daily = () => {
   ];
 
   return (
-    <div className="daily">
-      <div className="daily__top">
-        <h2 className="header--title">Daily</h2>
-        <span className="header--text">
-          We run all macro benchmark workloads against the <i>main</i> branch every day.
-          This is done to ensure the consistency of the results over time on <i>main</i>.
-          On this page, you can find graphs that show you the results of all five macro
-          benchmark workload over the last 30 days.
-        </span>
-        <br></br>
-        <span className="header--text">
-          Click on a macro benchmark workload to see all the results for that workload.
-        </span>
-      </div>
-      <figure className="line"></figure>
-      {errorDailySummary || dailyError ? (
-        <div className="apiError">{dailyError}</div>
-      ) : isLoadingDailySummary ? (
-        <div className="loadingSpinner">
+    <>
+      <Hero />
+
+      <figure className="p-page w-full">
+        <div className="border-front border" />
+      </figure>
+
```

Comment:
I feel like we should reduce a little the size of the vertical space between the header and the title and the rest of the daily page. This also applies to the other pages.

Reply:
I have changed the heights on most pages, please tell me if this is better

Reply:
It looks better! thanks a lot